### PR TITLE
Feature/ganesh

### PR DESCRIPTION
**Pull request by SekarGaneshAppDev**

### CODE REVIEW

File: src/views/users/Users.jsx [hunk 1]
```diff
@@ -297,15 +296,26 @@ class Users extends React.Component {
...
     ];
   }
+
   //User's Vehicle List
-  userVechileFun(e) {
-    var user_id = parseInt(e.currentTarget.dataset.user_id);
-    var data = { UserId: user_id };
-    this.props.getUserVehicleDetails(data);
+  expand_row(row) {
+    var expanded = { ...this.state.expanded };
```
Comment (varunsapp) [inline src/views/users/Users.jsx]: Expanded is initialized as null, cant spread null value error is possible.

Destructure state and assign empty object as default value

File: src/views/users/Users.jsx [hunk 2]
```diff
@@ -486,6 +496,14 @@ class Users extends React.Component {
                   <h3 className="mb-0">Users List</h3>
                 </CardHeader>
                 <ReactTable
+                  expanded={this.state.expanded}
+                  getTdProps={(state, rowInfo) => {
+                    return {
```
Comment (varunsapp) [inline src/views/users/Users.jsx:506]: Destructure expanded in render method

**Review by varunsapp — CHANGES_REQUESTED** [review]

File: src/views/users/Users.jsx (hunk 1 above)
Comment (SekarGaneshAppDev) [reply src/views/users/Users.jsx]: Fixed

File: src/views/users/Users.jsx (hunk 2 above)
Comment (SekarGaneshAppDev) [reply src/views/users/Users.jsx:506]: Fixed

**Review by varunsapp — APPROVED** [review]

**Pull request merged**